document/core/exec/numerics.rst

```diff
@@ -303,6 +457,18 @@ The integer result of predicates -- i.e., :ref:`tests <syntax-testop>` and :ref:
    it holds that :math:`i_1 = i_2\cdot\idivs(i_1, i_2) + \irems(i_1, i_2)`.
 
 
+.. _op-inot:
```

Comment:
5. A couple of new numeric ops are defined that are pretty straightforward. We need this as SIMD has some numeric instructions, like or and andnot, that are not yet defined.